build.nix

```diff
@@ -0,0 +1,40 @@
+{ lib, rustPlatform, rust, buildInputs, nativeBuildInputs, ... }:
+
+let
+  pname = "florestad";
+  version = "0.1.0";
+
+  buildRustPackage = rustPlatform.buildRustPackage.override {
+    rustc = rust;
+    cargo = rust;
+  };
+in
+
+buildRustPackage {
+  inherit pname version;
+
+  doCheck = false;
+
+  src = ./.;
+
+  cargoLock = {
+    lockFile = ./Cargo.lock;
+    outputHashes = {
+      "bitcoin-0.31.0" = "sha256-2v87cb+Wd4OhgDG45Za3bxumaY7QKdw7nKuqKrmtHMs=";
+    };
+  };
+
+  inherit buildInputs nativeBuildInputs;
+
+  # Override the Rust compiler used
+  rustc = "${rust}/bin/rustc";
+  cargo = "${rust}/bin/cargo";
```

Comment:
is this really needed?

Reply:
Yes, to ensure that the flake are using 1.74 rust